cp3-networkpolicy/install_networkpolicy.sh

```diff
@@ -197,7 +197,7 @@ function install_networkpolicy() {
     
     for policyfile in `ls -1 ${BASE_DIR}/services/*.yaml`; do
         info "Installing `basename ${policyfile}` ..."
-        cat ${policyfile} | sed -e "s/csNamespace/${CS_NAMESPACE}/g" | oc apply -f -
```

Comment:
The policies in `services` folder will be applied to pods in operandNS(CS_NAMESPACE for services folder). However in case we want to allow the ingress specifically from operatorNS(OPERATORS_NAMESPACE), we need to have two different placeholders.

Reply:
yup, got it